sklearn/feature_selection/_mutual_info.py

```diff
@@ -201,11 +202,13 @@ def _iterate_columns(X, columns=None):
 def _estimate_mi(
     X,
     y,
+    *,
```

Comment:
I used keyword arguments to make the code more readable in the `_estimate_mi` function call and also use keyword-only arguments in the `_estimate_mi` definition.

I guess that's fine since `_estimate_mi` is private. @glemaitre do you agree?

Reply:
Yes I do agrre.